src/components/blocks/newsletter-signup/newsletter-signup.html

```diff
@@ -1,3 +1,7 @@
+{% if modifier === "--footer" %}
+<div class="background-color: black; height: 100vh;">
```

Comment:
this can't be doing anything, so I guess just remove the dodgy class attribute

Reply:
ah whoops, have deleted it!

Reply:
Boom! :D